subprojects/android/android.gradle

```diff
@@ -3,8 +3,31 @@ description = "Mockito for Android"
 apply from: "$rootDir/gradle/java-library.gradle"
 
 dependencies {
-    compile project.rootProject
+    compile(project.rootProject) {
+        exclude group: 'org.objenesis', module: 'objenesis'
+    }
     compile libraries.bytebuddyandroid
+    compile(libraries.objenesis2) {
+        version {
+            strictly '2.6'
```

Comment:
Can we make this version `[2.6, 3.0[`, so that we also allow all Objenesis 2.x versions?

Reply:
Updated versions and test locally. It works as expected.